packages/playwright-core/src/remote/playwrightConnection.ts

```diff
@@ -245,6 +250,18 @@ export class PlaywrightConnection {
     debugLogger.log('server', `[${this._id}] finished cleanup`);
   }
 
+  private logServerMetadata(message: object, direction: 'SEND' | 'RECV') {
+    const serverLogMetadata: { [key: string]: any } = {
+      timestamp: Date.now(),
+      payloadSizeInBytes: Buffer.byteLength(JSON.stringify(message), 'utf-8')
```

Comment:
Let's pass `messageString` along with message, because we always have the stringified version at the call sites.